docs/generics/generics-and-reference-capabilities.md

```diff
@@ -50,7 +50,13 @@ That compiles and runs, so `ref` is valid now. The real test though is `iso`. Le
 This fails to compile. The first error is:
 
 ```error
---8<-- "generics-and-reference-capabilities-foo-iso-error-message.txt"
+main.pony:5:8: right side must be a subtype of left side
+    _c = c
+       ^
+    Info:
+    main.pony:4:17: String iso! is not a subtype of String iso: iso! is not a subtype of iso
+      new create(c: String iso) =>
+                ^
```

Comment:
This looks unrelated to the change?

Reply:
Kinda. This was because the PR was created alongside previous PRs and when we reverted non-pony examples and cherry-picked them, not everything looked exactly pretty in the commit history 😅